src/control/cmd/daos/filesystem.go

```diff
@@ -105,6 +107,10 @@ func (cmd *fsCopyCmd) Execute(_ []string) error {
 	cmd.Infof("    Files:       %d", ap.fs_copy_stats.num_files)
 	cmd.Infof("    Links:       %d", ap.fs_copy_stats.num_links)
 
+	if ap.fs_copy_stats.num_chmod_enotsup > 0 {
```

Comment:
nit: comparison with bool should be "== 1"

Reply:
num_chmod_enotsup isn't a bool?

Reply:
sorry, my mistake